wandb/data_types.py

```diff
@@ -1559,6 +1559,16 @@ def seq_to_json(cls, images, run, key, step):
         width, height = images[0]._image.size
         format = jsons[0]["format"]
 
+        def size_equals_image(image):
+            img_width, img_height = image._image.size
+            return img_width == width and img_height == height
+
+        sizes_match = all(size_equals_image(img) for img in images)
+        if not sizes_match:
+            logging.warning(
+                "Images sizes do not match. This will causes images to be display incorrectly in the UI."
```

Comment:
Is there anything we can do to make it so the images do display correctly? I am thinking about cases where the user does want to display/log different size images. Consider a model that attempts to improve the resolution of an image (eg. https://www.pixelmator.com/blog/2019/12/17/all-about-the-new-ml-super-resolution-feature-in-pixelmator-pro/ or maybe a model that extends or crops an image). I am not sure when this `seq_to_json` method is used (and if it would apply to such cases), but if so, we might want to handle this more dynamically in the UI.

Reply:
In the old days we used to sprite these images together so it was a strict requirement.   I didnt think that is what was happening any more with the latest CLI (0.10.x+)

Reply:
Adding support for multi-size images is not an easy task. I've already attempted to make changes to the frontend and CLI to get this to work correctly, but sadly modify the MediaPanel is difficult. We will soon replace the Media Panel with the new Data vis which will add support for this. As for the time being, I think a warning would be appropriate.

Reply:
Makes sense - thanks for the additional details.